src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -85,7 +85,7 @@ export const AdvancedSearchPage = () => {
     sort: params.get(ResultParam.Sort) as SortOrder | null,
     title: params.get(ResultParam.Title),
     excludeSubunits,
-    topLevelOrganization: excludeSubunits && !!unitId ? null : topLevelOrganizationId,
+    topLevelOrganization: topLevelOrganizationId,
     unit: unitId,
```

Comment:
Tror dette burde være nor sånt, ref diskusjon på slack: 
```suggestion
    unit: unitId || topLevelOrganizationId,
```

Reply:
i settingen her er vi alltid interessert i unit, aldri topLevelOrg